test/e2e/condition.py

```diff
@@ -0,0 +1,146 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
```

Comment:
Can all of this file go into `acktest` instead? :)

Reply:
https://github.com/aws-controllers-k8s/test-infra/pull/166